bin/circleci-update-yarn-lock

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+git log --name-status HEAD^..HEAD | grep "fix(package): update" || { echo ">> not an update commit"; exit 0; }
+
+echo ">> running yarn install --pure-lockfile"
+yarn install --no-progress --pure-lockfile
+
+yarn check && { echo ">> yarn check passed, yarn.lock does not need to update"; exit 0; }
+
+echo ">> running yarn install --force"
+yarn install --force --no-progress
+
+name="greenkeeper[bot]"
+email="greenkeeper[bot]@users.noreply.github.com"
+
+git config user.name "$name"
+git config user.email "$email"
+
+echo ">> git config $name <$email>"
```

Comment:
nit: avoid var interpolation in `echo`. Use `printf` here if you want to log these (though since they're hard-coded I also personally think you could just drop this message.